cypress/support/index.js

```diff
@@ -39,6 +41,24 @@ Cypress.Commands.overwrite(
   ),
 );
 
+Cypress.Commands.add('radioButtonComponent', (radioButtonLabel) => {
+  radioButtonComponent().each(($el) => {
+    const labelText = $el.find('label').text();
+    if (labelText.includes(radioButtonLabel)) {
+      cy.wrap($el);
+    }
+  });
+});
+
+Cypress.Commands.add('radioButton', (radioButtonLabel, findBy) => {
+  radioButtonComponent().each(($el) => {
+    const labelText = $el.find('label').text();
+    if (labelText.includes(radioButtonLabel)) {
+      cy.wrap($el).find(findBy);
```

Comment:
could we try to use `cy.('element').as('storredElement')`?

Reply:
We can't use an alias and remove `find` from here, because when I moved `find` to `radio-button-steps` the`find` function return 3 element. When the same `find` is inside of `radioButtonComponent()` the `find` return only one element because `each` method handle it